### PR TITLE
xetex: add page

### LINKED ISSUE #6318: page request: xetex
https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=xetex

### PR DESCRIPTION
**Pull request by chrissxYT**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

basically a copy-paste from `pdflatex`, because the basic usage is the same. closes #6318

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Is compile the correct wording, I would use create.

Compile to me is more like compiling C code

File: pages/common/xetex.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Compile a PDF document:
+
+`xetex {{source.tex}}`
+
+- Compile a PDF document specifying an output directory:
+
+`xetex -output-directory={{path/to/directory}} {{source.tex}}`
+
+- Compile a PDF document, halting on each error:
```
Comment (CleanMachine1) [inline pages/common/xetex.md:14]:
```suggestion
- Compile a PDF document, stopping on each error:
```

Does this exit or just highlight?

Comment (chrissxYT) [reply pages/common/xetex.md:14]: It exits.

**Comment by chrissxYT** [comment]
> Is compile the correct wording, I would use create.

Creating sounds to me like it creates it out of nowhere, I would rather use convert.

> Compile to me is more like compiling C code

I can see that, but _serious computer science_ people call everything compiling. And that kinda makes sense, TeX really looks like a programming language, and it is one, when you look at some crazy macros. And `xetex` compiles TeX code to PDF.

File: pages/common/xetex.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/xetex.md:14]: Then here is a better alternative

```suggestion
- Compile a PDF document, exiting if errors occur:
```

**Comment by CleanMachine1** [comment]
Consider that TLDR is aimed towards beginners, regardless of the subject or page.

Convert is beginner language, however see what others think before making a change like that.

File: pages/common/xetex.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/xetex.md]: I think we should also apply this to `pdflatex`, `latex`, `pdftex` and `tex`.

**Comment by marchersimon** [comment]
@CleanMachine1 `compiling` is indeed the correct wording for TeX/LaTeX/... . All TeX engines are basically compilers.

File: pages/common/xetex.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# xetex
+
+> Compile a PDF document from XeTeX source files.
+> More information: <https://manned.org/xetex>.
```
Comment (marchersimon) [inline pages/common/xetex.md:4]: Remember to only use `manned.org` if there's nothing better available.
```suggestion
> More information: <https://www.tug.org/xetex/>.
```

File: pages/common/xetex.md (hunk 1 above)
Comment (chrissxYT) [reply pages/common/xetex.md]: That would have been my next PR, once this is merged.

**Review by marchersimon — APPROVED** [review]
Thanks @chrissxYT

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/xetex.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# xetex
+
+> Compile a PDF document from XeTeX source files.
+> More information: <https://www.tug.org/xetex/>.
+
+- Compile a PDF document:
+
+`xetex {{source.tex}}`
+
+- Compile a PDF document specifying an output directory:
```
Comment (CleanMachine1) [inline pages/common/xetex.md:10]:
```suggestion
- Compile a PDF document, specifying an output directory:
```

I feel this reads a little better.

Comment (marchersimon) [reply pages/common/xetex.md:10]: Yep, and it's also in the other pages.

**Review by navarroaxel — APPROVED** [review]
